### DISCUSSION TITLE
Pseudotile dwindle config option

### DISCUSSION THREAD

**Discussion by Aqa-Ib** [hyprwm/Hyprland#1104] [Q&A]
What is the pseudotile dwindle config option doing? If I set it to true, windows are not defaulting to pseudo mode.
Thank you so much for your attention. Regards.

**Comment by vaxerski** [comment]
you need to toggle it separately, the config option is a master switch.

**Comment by Aqa-Ib** [comment]
I enabled it like this:
```
dwindle {
    pseudotile=true
}
```
What am I doing wrong?
Thank you very much.

**Reply by vaxerski** [reply]
![image](https://user-images.githubusercontent.com/43317083/204097131-36d5cdda-08af-48b1-80ee-0acd363208d3.png)
https://wiki.hyprland.org/Configuring/Dispatchers/

Read the wiki.

**Comment by Aqa-Ib** [comment]
Hmmm I am missing something. It does not matter if I set the **pseudotile** option to true or false, the pseudo dispatcher is working the same. What is the point of the pseudotile option?

**Reply by vaxerski** [reply]
it's useless anyways, so I probably deprecated it.

**Comment by Aqa-Ib** [comment]
I see. Thank you again for your patience, vaxerski. Have a nice day.
